src/dotnetTemplate/Microsoft.PowerShell.PowerShellStandard.Template/Microsoft.PowerShell.PowerShellStandard.Template/.template.config/template.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft Corporation",
+  "classifications": [ "Common", "Console", "PowerShell", "Module" ],
+  "name": "PowerShell Standard Module",
+  "identity": "Microsoft.PowerShell.PowerShellStandard.Template",
+  "groupIdentity": "Microsoft.PowerShell.PowerShellStandard.Template",
+  "shortName": "psmodule",
```

Comment:
Does the casing affect how it's displayed?  If so, use `PSModule`

Reply:
All of the other templates use all lowercase. I don't think we should go against that trend. What do you think?